math/classical_sieve_of_eratosthenes.cpp

```diff
@@ -39,8 +38,10 @@ void print(int N) {
 }
 
 int main() {
+  fastio();
```

Comment:
syntax error `fastio()` is not defined

Reply:
@ayaankhan98 I have already removed it in the last commit